src/config.rs

```diff
@@ -26,15 +26,15 @@ pub struct Config {
 #[derive(Debug, Default, Serialize, Deserialize)]
 pub struct GeneralConfig {
```

Comment:
Any chance that we can make the sections (`[general]` and `[gpg]`) optional as well?